lib/widgets/action_sheet.dart

```diff
@@ -91,7 +91,11 @@ void _showActionSheet(
                           color: designVariables.bgContextMenu,
                           child: SingleChildScrollView(
                             padding: const EdgeInsets.symmetric(vertical: 8),
-                            child: MenuButtonsShape(buttons: optionButtons)))),
+                            child: Column(
```

Comment:
nit:

    action_sheet [nfc]: Support multiple button sections in context menu
    
    Some context menus in the Figma have multiple groups/sections of

s/context menu/action sheet/g